docroot/modules/custom/foia_request/src/Entity/FoiaRequestInterface.php

```diff
@@ -28,6 +28,11 @@
    */
   const STATUS_FAILED = 2;
 
+  /**
+   * Request pending file attachment virus scan.
+   */
+  const STATUS_SCAN = 3;
+
```

Comment:
I think we need to update the `getRequestStatus` and `setRequestStatus` doc blocks to include this new status.